src/servers/sapphire_zone/DebugCommand/DebugCommandHandler.cpp

```diff
@@ -768,6 +768,45 @@ void Core::DebugCommandHandler::instance( char* data, Entity::Player &player, bo
 
       instance->setVar( static_cast< uint8_t >( index ), static_cast< uint8_t >( value ) );
    }
+   else if( subCommand == "objupdate" )
+   {
+      uint32_t instanceId;
```

Comment:
can't we just grab instance ID from the player? seems like it'd get uuuh repetitive